### PR TITLE
Add support for TwoWindingTransformers in expert filters

### PR DESCRIPTION
**Pull request by ayolab**

### CODE REVIEW

**Comment by lilian-houdelet** [comment]
It is possible to change the French label from "Transfos" to "Transformateur" ?

File: src/components/dialogs/filter/expert/expert-filter-constants.ts [hunk 1]
```diff
@@ -239,6 +243,82 @@ export const FIELDS_OPTIONS = {
         dataType: DataType.NUMBER,
         inputType: 'number',
     },
+    NOMINAL_VOLTAGE_1: {
+        name: FieldType.NOMINAL_V_1,
+        label: 'nominalVoltage1',
```
Comment (AAJELLAL) [inline src/components/dialogs/filter/expert/expert-filter-constants.ts]: Shouldn't be something like nominalVoltageOr?

File: src/components/dialogs/filter/expert/expert-filter-constants.ts [hunk 2]
```diff
@@ -239,6 +243,82 @@ export const FIELDS_OPTIONS = {
...
+        dataType: DataType.NUMBER,
+        inputType: 'number',
+    },
+    NOMINAL_VOLTAGE_2: {
+        name: FieldType.NOMINAL_V_2,
```
Comment (AAJELLAL) [inline src/components/dialogs/filter/expert/expert-filter-constants.ts]: The same here: nominalVoltageEx?

**Review by lilian-houdelet — APPROVED** [review]

**Review by AAJELLAL — APPROVED** [review]
Test and code OK.

**Pull request merged**